samples/scaleout/Version_6/Shared/ConfigErrorQueue.cs

```diff
@@ -0,0 +1,13 @@
+﻿using NServiceBus.Config;
+using NServiceBus.Config.ConfigurationSource;
+
+class ConfigErrorQueue : IProvideConfiguration<MessageForwardingInCaseOfFaultConfig>
```

Comment:
use the code api instead SendFailedMessagesTo http://docs.particular.net/nservicebus/errors/#errorwithcode-v6-pre